src/callbacks/amr.jl

```diff
@@ -132,17 +133,17 @@ end
 
 
 """
-    IndicatorThreeLevel(semi, indicator; base_level=1,
-                                         med_level=base_level, med_threshold=0.0,
-                                         max_level=base_level, max_threshold=1.0)
+    ControllerThreeLevel(semi, indicator; base_level=1,
```

Comment:
TBH, I do not like the name `Controller` here, as it gives no indication of what is being controlled. I have a similar beef with `Indicator`, but since that's used for multiple purposes, I was not able to come up with a better name. However, here I'd suggest to call the AMR Controllers `AmrController` (or `AMRController`). What do you think @ranocha?